src/inference/include/openvino/runtime/compiled_model.hpp

```diff
@@ -141,7 +137,7 @@ class OPENVINO_RUNTIME_API CompiledModel {
      * @return Compiled model output.
      * @note The method throws ov::Exception if output with the specified tensor name @p tensor_name is not found.
      */
-    ov::Output<const ov::Node> output(const std::string& tensor_name) const;
+    const ov::Output<const ov::Node>& output(const std::string& tensor_name) const;
```

Comment:
does it break source compatibility?

Reply:
In general it shouldn't affect customer.
In case of:
```
 auto output = compiled_model.output();
 ov::Output<const ov::Node> output = compiled_model.output();
```
He will get a copy of the object.